src/openrct2/EditorObjectSelectionSession.cpp

```diff
@@ -408,16 +412,15 @@ static void set_object_selection_error(uint8_t is_master_object, rct_string_id e
  *
  *  rct2: 0x006AB54F
  */
-int32_t window_editor_object_selection_select_object(uint8_t bh, int32_t flags, const rct_object_entry* entry)
+bool window_editor_object_selection_select_object(uint8_t isMasterObject, int32_t flags, const ObjectRepositoryItem* item)
```

Comment:
What is a "master object"? Seems weird to have something that "isSomething" but isn't a boolean.

Reply:
I took the name from the underlying function. I'm not 100% sure it is correct, which is why I didn't make any changes to the data type yet.

Reply:
Ah, sure, I'm not questioning, I'm trying to understand because I have no clue hahaha